sotodlib/site_pipeline/update_obsdb.py

```diff
@@ -219,10 +219,25 @@ def main(config: str,
             except KeyError:
                 logger.error("No telescope key in index file or error with lat_tube_list")
                 very_clean["telescope_flavor"] = "unknown"
-            stream_ids = index.pop("stream_ids")
-            if stream_ids is not None:
+            
+            #Stream_ids and wafers
+            try:
+                stream_ids = index["stream_ids"]
                 bookcartobsdb.add_obs_columns(["wafer_count int"])
                 very_clean["wafer_count"] = len(stream_ids)
+                wafer_slots = index["wafer_slots"]
+                if len(wafer_slots) < len(stream_ids):
+                    logger.error("Missing info on some stream_ids")
```

Comment:
just want to confirm it's ok to proceed to add columns despite missing info on some stream_ids here?

Reply:
Yeah I should add a continue in that case.